### PR TITLE
Attack portal location calculator

### PR DESCRIPTION
**Pull request by ryytz**
Created range_utility and location_calculator
Fixed matan's wrappers

### CODE REVIEW

File: GoblinSabotageSquad/MyBot.py [hunk 1]
```diff
@@ -27,31 +27,6 @@ def calculate_third_portal_location(self):
         col = int((self.my_castle_location.col + self.enemy_castle_location.col) / 2)
         return Location(row=row, col=col)
 
-
```
Comment (matangn13) [inline GoblinSabotageSquad/MyBot.py:28]: Why is this deleted? is it pasted to another place?

Comment (ryytz) [reply GoblinSabotageSquad/MyBot.py:28]: It is deleted because a different strategy of defense is installed refer defense location calc pull request
